docs/getting-started/typical-workflow-cli-user.md

```diff
@@ -222,10 +222,10 @@ Once the session is ready, you have three options to interact with it:
 - ssh from your local
 - ssh + VSCode
 
-Let's login to the Session via SSH.
+Let's login to the Session via SSH using the previously added key.
 
 ```yaml
-grid session ssh resnet-debugging
+grid session ssh resnet-debugging -- -i ~/.ssh/grid_ssh_creds
```

Comment:
How bout just adding the key to ssh agent?

Reply:
Yea, that's better idea. Gonna update the docs in a second.